clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/jobhandling/CommandWrapper.java

```diff
@@ -35,7 +35,7 @@ public class CommandWrapper {
   private final int maxRetries;
 
   public CommandWrapper(
-      final FinalCommandStep<Void> command,
+      final FinalCommandStep<?> command,
```

Comment:
Didn't see a need to force specific generic type usage as the value is not used internally. We could change it when this is actually needed, till then the wildcard type should be suffice.